MediaBrowser.MediaEncoding/Encoder/EncoderValidator.cs

```diff
@@ -123,12 +120,66 @@ private enum Codec
 
         public static Version MaxVersion { get; } = null;
 
-        public bool ValidateVersion()
+        /// <summary>
+        /// Returns a value indicating which version of the encoder more suitable.
+        /// </summary>
+        /// <param name="source">Source encoding version.</param>
+        /// <param name="destination">Destination encoding version.</param>
+        /// <returns>-1 if source is better, 1 if destination is better.</returns>
```

Comment:
I think `bool` would fit better here
Maybe `true` if the new version is better and `false` if the current version is the best version?

Reply:
changed